SU-SOE/stanford_soe_helper
# SOE-2334: #2 padding

# READY FOR REVIEW

# Summary
- [Kerri Responsive tweaks 03](https://docs.google.com/spreadsheets/d/1O7VSJD25zNH722Fbg4XcHl3M6KkAZAGKtzRCkU9M45c/edit#gid=0)

This is for lines 1-11.

# Needed By (Date)
- Sprint end

# Criticality
- Fixed on prod, need to put into code.

# Steps to Test
- Switch to this branch
- Verify against screen shots from [documentation](https://docs.google.com/spreadsheets/d/1O7VSJD25zNH722Fbg4XcHl3M6KkAZAGKtzRCkU9M45c/edit#gid=0). 

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2334

## Related PRs
This re-implements commits from: https://github.com/SU-SOE/stanford_soe_helper/pull/131

## More Information

## Folks to notify
Kerri


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
